File: lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/models/search/KgSearchFilters.scala
package io.github.tetherlessworld.mcsapps.lib.kg.models.search

import io.github.tetherlessworld.mcsapps.lib.kg.models.StringFacetFilter

final case class KgSearchFilters(
                                  sourceIds: Option[StringFacetFilter]
                                )

File: gui/src/shared/models/kg/search/KgSearchFilters.ts
import {StringFilter} from "shared/models/kg/search/StringFilter";
import {KgSearchResultTypeFilter} from "shared/models/kg/search/KgSearchResultTypeFilter";

export interface KgSearchFilters {
  sourceIds?: StringFilter | null;
  types?: KgSearchResultTypeFilter | null;
}
